src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/MessageStrings.resx

```diff
@@ -144,6 +144,9 @@
   <data name="ForceOptionDesc" xml:space="preserve">
     <value>Specify whether to overwrite existing files.</value>
   </data>
+  <data name="InstallEfPackages" xml:space="preserve">
+    <value>To scaffold controllers and views using models please install Entity Framework core packages and try again: {0}</value>
```

Comment:
We need two packages to be installed, `Microsoft.EntityFrameworkCore.Design` and `Microsoft.EntityFrameworkCore.SqlServer` 

The message displayed to the user will have both these ids. 
Are you suggesting, we only show the one that is missing?